api/twilio-api.md

```diff
@@ -52,27 +52,27 @@ TwilioQuest is an interactive tutorial where users can learn how to use Twilio t
 	- **Exercise:** Create a POST request using JSON and look at the Response.
 
 - Example:
-![alt text](https://github.com/Techtonica/curriculum/blob/master/api/Screen%20Shot%202018-10-08%20at%207.15.15%20PM.png)
+![alt text](./Screen%20Shot%202018-10-08%20at%207.15.15%20PM.png)
```

Comment:
👍

Reply:
what should i do in api/twillo-api.md?

Reply:
Nothing just +1 this find. I forgot about the images.